specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json

```diff
@@ -6709,7 +6703,7 @@
         "NotSpecified",
         "AS2",
         "X12",
-        "Edifact"
+        "EDIFACT"
```

Comment:
Wont this be a breaking change? If so, it will impact the SDK users. Are you really ok with this

Reply:
I'm still in the process of getting a new (major breaking) version of the SDK out. This was caused by the deserialization of a response from our API in the SDK coming back as all caps. I'll just put this back to being not modeled as a string. This enum is unlikely to be added to